settings.gradle

```diff
@@ -153,3 +153,6 @@ if (extraProjects.exists()) {
     addSubProjects('', extraProjectDir)
   }
 }
+include 'modules:throttling'
```

Comment:
```
'addSubProjects('', new File(rootProject.projectDir, 'modules'))
```
is present, so can you please remove this ?

Reply:
Yes this not required will delete this.